static/app/components/arithmeticInput/parser.tsx

```diff
@@ -26,13 +29,27 @@ export class Operation {
   }
 }
 
+class Primary {
```

Comment:
Honestly not a huge fan of this name 😓, but the best I could think of so I'm open to better suggestions

Reply:
One suggestion I have here is to rename `Term` -> `Expression` and use `Term` here